shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -0,0 +1,723 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+from functools import wraps
+
+from odoo.osv.expression import AND
+from odoo.tools import float_compare
+
+from odoo.addons.base_rest.components.service import to_int
+from odoo.addons.component.core import Component
+from odoo.addons.component.exception import NoComponentError
+
+_logger = logging.getLogger("shopfloor.services.single_product_transfer")
+
+
+def with_savepoint(meth):
```

Comment:
```suggestion
def with_savepoint(method):
```
The abreviation is not neede imp, except if `method` is a reserved python word ?